src/plugins/installer.py

```diff
@@ -11,6 +11,12 @@
 
 
 class AbstractPluginInstaller:
+    """A class that is used to handle plugin installation.
+    Any class subclassing it may overwrite any public method except :py:func:`install`.
```

Comment:
The reference to the method is probably not that useful if we don't add this file to the sphinx docs